portality/static/widget/fq_widget_depends_compiled.js

```diff
@@ -3269,372 +3269,3 @@ function journal_toc_id(journal) {
 
     return toc_id;
 }
-
```

Comment:
what's up with the big delete of fv call handling functions below, are they just not used by the widget's faceview config?